python/nav/web/report/views.py

```diff
@@ -409,7 +409,7 @@ def find_page_range(page_number, page_range, visible_pages=5):
     if length <= visible_pages:
         return page_range
 
-    padding = visible_pages / 2
+    padding = visible_pages // 2
```

Comment:
Luckily works both in python 2.7 and any 3.x

Reply:
I know.